packages/overlay/src/index.tsx

```diff
@@ -57,6 +72,7 @@ export const Overlay: React.FC<OverlayProps> = ({
   shouldCloseOnClick = true,
   shouldCloseOnEscapePress = true,
   preventBodyScrolling = false,
+  verticalAlign = 'stretch',
```

Comment:
Current default is `center`, isn't it?

Reply:
Current resulting behavior is `stretch`, which basically was an incentive for creating this PR :D